webviz_subsurface/plugins/_multiple_regression.py

```diff
@@ -0,0 +1,1129 @@
+                [
+                    html.Div(
+                        "Max number of parameters",
+                        style={"display": "inline-block", "margin-right": "10px"},
+                    ),
+                    dcc.Dropdown(
+                        id=self.uuid("max-params"),
+                        options=[
+                            {"label": val, "value": val}
+                            for val in range(1, min(10, len(self.parameterdf.columns)))
+                        ],
+                        clearable=False,
+                        value=3,
+                    ),
+                ]
+            ),
+            html.Div(
+                [
+                    html.Div(
+                        "Force in", style={"display": "inline-block", "margin-right": "10px"},
+                    ),
+                    dcc.Dropdown(
+                        id=self.uuid("force-in"),
+                        clearable=True,
+                        multi=True,
+                        placeholder="Select parameters to force in",
+                        value=[],
+                        style={"marginBottom": "20px"},
+                    ),
+                ]
+            ),
+            html.Div(
+                style={"display": "grid"},
+                children=[
+                    html.Button(
+                        id=self.uuid("submit-button"),
+                        children="Update model",
+                        style={
+                            "background-color": "LightGray",
+                            "cursor": "not-allowed",
+                            "border": "none",
+                        },
+                        disabled=True,
+                    )
+                ],
+            ),
+        ]
+
+    @property
+    def layout(self):
+        """ Main layout """
+        return wcc.FlexBox(
+            id=self.uuid("layout"),
+            children=[
+                html.Div(style={"flex": 1}, children=self.control_layout),
+                html.Div(
+                    style={"flex": 3},
+                    children=[
+                        html.Div(
+                            id=self.uuid("page-title"),
+                            style={
+                                "textAlign": "center",
+                                "display": "grid",
+                                "font-weight": "bold",
+                                "fontSize": "1.3em",
+                            },
+                        ),
+                        html.Div(children=[wcc.Graph(id=self.uuid("p-values-plot"))]),
+                        html.Div(children=[wcc.Graph(id=self.uuid("coefficient-plot"))]),
+                        html.Label(
+                            "Table of parameters and their corresponding p-values",
+                            style={
+                                "fontSize": ".925em",
+                                "color": self.colors["default text"],
```

Comment:
Think this can be removed, we already have CSS-code that should be able to style the labels with the right color.
```suggestion
```